src/styles/grade.scss

```diff
@@ -0,0 +1,26 @@
+.grade {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px 24px;
+  font-size: 10px;
+  line-height: 14px;
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+
+    &-star {
+      padding-right: 4px;
+      background-image: url('../images/star.svg');
+      width: 16px;
+      height: 16px;
+      background-repeat: no-repeat;
```

Comment:
The width and height of the `.grade__stars-star` can be extracted to a variable in `variables.scss` for better maintainability and to avoid magic numbers in the code. You can then use this variable here and in any other place where this size is needed.